.travis.yml

```diff
@@ -25,6 +25,7 @@ matrix:
       env:
         - JDK='Oracle JDK 8'
         - PYTHON=3.5 PYTHONWARNINGS=ignore
+        - RAY_USE_CMAKE=1
```

Comment:
Don't be fooled, this will go away pretty soon, we don't intend keeping to support two build systems for long!

Reply:
Why introduce the flag?

Reply:
Because cmake is used for building the wheels and in the valgrind build (to do things a bit more incrementally).